criu/log.c

```diff
@@ -199,8 +199,8 @@ void flush_early_log_buffer(int fd)
 		}
 		pos += hdr->len;
 	}
-	if (early_log_buf_off)
-		pr_warn("The early log isn't empty\n");
+	if (early_log_buf_off == EARLY_LOG_BUF_LEN)
```

Comment:
@0x7f454c46 , I'm a bit late, but please, drop a couple of lines about early log somewhere on the wiki. E.g. [here](https://criu.org/Logging) or a new "under-the-hood" article. Thanks :)

Reply:
done